### PR TITLE
Milestone friendships v1

### PR DESCRIPTION
**Pull request by ABDELLANI-Youcef**
## Milestone friendships v1
In this milestone we make:
- A friendship model, associations, and migration;
- A friendship controller and view;
- The possibility to send a friendship invitation and to accept or decline this invitation;
- A spec tests.

### CODE REVIEW

**Comment by ABDELLANI-Youcef** [comment]
We made a linter check in the local environment and everything was fine, but in Github, the linter tests didn't start.

File: config/database.yml [hunk 1]
```diff
@@ -19,11 +19,12 @@ default: &default
   encoding: unicode
   # For details on connection pooling, see Rails configuration guide
   # http://guides.rubyonrails.org/configuring.html#database-pooling
-  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  pool: 5
 
 development:
   <<: *default
   database: ror_social_scaffold_development
+  username: postgres
```
Comment (lekegitrepo) [inline config/database.yml]: - **_Remove `username: postgres` it affects the migration on the other computer_**

File: config/database.yml [hunk 2]
```diff
@@ -58,6 +59,7 @@ development:
 test:
   <<: *default
   database: ror_social_scaffold_test
+  username: postgres
```
Comment (lekegitrepo) [inline config/database.yml]: - **_Remove `username: postgres` it affects the migration on the other computer_**

**Review by lekegitrepo — CHANGES_REQUESTED** [review]
## Status: Request changes :recycle:
### Hi @ABDELLANI-Youcef and @Ceci007 :clap: :clap:
#### Well done :muscle:  You have done a good job so far. However, there some issues you need to fix before this project can be approved.

- **_Enable Linter_**

- **_Posts timeline should include only the posts written by logged-in user and their friends ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/final_project/milestone5/friends_timeline.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/49736080/))_**

#### Optional:
- **_I think it will improve the user experience if the accept and reject buttons are next to the appropriate user(in "ALL USERS") in case of the friend request instead of click on the "FRIEND REQUESTS" link before a user can accept or reject a friend request_**

**_Fix these and submit them for review._**

#### _Reviewed by Temitayo Adeleke_

**Comment by ABDELLANI-Youcef** [comment]
Hello, could you help us to enable linter on GitHub? In fact, we could run the linter in the local environment (our laptops) but when we push the project on GitHub we found that the tests are not running even though we set the linter all over again.

**Review by voscarmv — CHANGES_REQUESTED** [review]
Hi, team. Great job applying the necessary changes requested by the previous TSE. However, you still must fix your linters somehow.

I find this to be a very strange bug, because I've actually created a copy of your repository to see what the problem was, but in my case, the linters seem to be working perfectly.

Here's my version of your PR for future reference: https://github.com/voscarmv/tse_ror_test/pull/1

Here are a few suggestions to fix this issue:

1. Try modifying the linter .yml files slightly, and then pushing, to see if this activates the GitHub automated checks.
2. If all else fails, contact me through Slack. I might be able to provide you further technical support.

All the rest of the project requirements seem to be in order. This is the one last issue remaining.

**Pull request closed without merging**